windows/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ add_library(hidapi_winapi
 )
 target_link_libraries(hidapi_winapi
     PUBLIC hidapi_include
-    PRIVATE setupapi
+    PRIVATE cfgmgr32 hid
```

Comment:
And with this change you will break the build for everywone who don't have WDK installed (which is a large amount of users).

To have a support of DDK builds, it has to be done properly, e.g. a CMake option, and link against `hid.lib` only when the user explicitly needs to.

Reply:
Just to be clear: DDK support is out of scope of this PR.
And it would require a lot more than I've mentioned in the previous comment.

Reply:
Alright. I'll do another PR for that.